common/io/ExtendedSerial.cpp

```diff
@@ -32,6 +32,10 @@
 #ifdef HAVE_STROPTS_H
 // this provides ioctl() definition without conflicting with asm/termios.h
 #include <stropts.h>
+#else
+// this provides a minimal ioctl() definition for modern systems
+// which don't have <stropts.h> anymore
+#include <ola/io/IOCtl.h>
 #endif  // HAVE_STROPTS_H
 
 #ifdef HAVE_ASM_TERMIOS_H
```

Comment:
Which would suggest it's actually this bit that's breaking

Reply:
I don't think it is - `./configure` reports that both `termios2.h` and `asm/termios.h` are present and usable, whereas `stropts.h` is not; and in any case, if it were this, then my "fix" wouldn't actually fix anything.. so unless something really non-obvious is happening I don't think the issue is here.